openreview/venue/webfield/programChairsWebfield.js

```diff
@@ -28,7 +41,7 @@ return {
     areaChairName: domain.content.area_chairs_name?.value,
     reviewerName: domain.content.reviewers_name.value,
     anonReviewerName: domain.content.reviewers_anon_name.value,
-    anonAreaChairName: domain.content.area_chairs_anon_name.value,
+    anonAreaChairName: domain.content.area_chairs_anon_name?.value,
```

Comment:
It might be good to use `?` for all values from `domain.content`.

Reply:
done